project/Publish.scala

```diff
@@ -21,42 +21,39 @@ import xerial.sbt.Sonatype.autoImport.sonatypeProfileName
 
 object Publish extends AutoPlugin {
 
-  val defaultPublishTo = settingKey[File]("Default publish directory")
-
   override def trigger = allRequirements
 
   override lazy val projectSettings = Seq(
-    publishTo := Some(akkaPublishTo.value),
+    publishTo := Some(pekkoPublishTo.value),
     publishRsyncHost := "akkarepo@gustav.akka.io",
-    credentials ++= akkaCredentials,
-    organizationName := "Lightbend Inc.",
-    organizationHomepage := Some(url("https://www.lightbend.com")),
-    sonatypeProfileName := "com.typesafe",
-    startYear := Some(2009),
+    credentials ++= pekkoCredentials,
+    organizationName := "Apache Inc.",
```

Comment:
@pjfanning Do all of these settings look correct to you?

Reply:
Apache Software Foundation - I think Apache Inc is a private and unrelated company